fhir-search/src/test/java/com/ibm/fhir/search/test/InclusionParameterParseTest.java

```diff
@@ -34,7 +34,7 @@
 /**
  * This TestNG test class contains methods that test the parsing of search result inclusion parameters (_include and
  * _revinclude) in the SearchUtil class.
- * 
+ *
  * @author markd
  * @author pbastide
```

Comment:
Once we went open source, we decided to remove @author tags from the code, but rather than all at once we just remove them as we find them.
```suggestion
```

Reply:
done